### DISCUSSION TITLE
How to make a homework solution visible ONLY to the classroom students?

### DISCUSSION THREAD

**Discussion by taiya** [community/community#43348] [GitHub Education] [answered]
How to make a homework solution visible ONLY to the classroom students?
If I make the repo public within the organization, it will be public to the world, no?
Is the only way to create a "dummy" assignment with the solutions included?
(this seems silly)

Thanks for the feedback.

p.s. I was trying to login on `https://education.github.community`, but I get:
```
Login Error
There is a problem with your account. Please contact the site's administrator.
```
Is that just a bad error message?

**Comment by taiya** [comment, marked as answer]
Had a discussion with a Github Classroom team representative (she cannot reply here).
TL;DR: not quite possible yet.

The suggestion was to use "Teams" and have a private repo that only a particular team under the (classroom) organization can read.
However, team members in Teams can only added one-by-one -- pretty painful for large classes! There is an open ticket for having "auto-teams" that is being worked on (i.e. all students added to a "students" team), but this is not ready yet, and there is no ETA.

Another suggestion was to use gists, and share a "secret" gist (i.e. link only) with students.

**Comment by asglover** [comment]
Is this still the recommended workflow?
